src/views/admin/KlasseCreationView.vue

```diff
@@ -0,0 +1,346 @@
+        :closable="false"
+        :text="$t('admin.klasse.klasseCreateErrorText')"
+        :showButton="true"
+        :buttonText="$t('admin.klasse.backToCreateKlasse')"
+        :buttonAction="navigateBackToKlasseForm"
+      />
+
+      <!-- The form to create a new Klasse -->
+      <template v-if="!organisationStore.createdOrganisation && !organisationStore.errorCode">
+        <FormWrapper
+          :confirmUnsavedChangesAction="handleConfirmUnsavedChanges"
+          :createButtonLabel="$t('admin.klasse.create')"
+          :discardButtonLabel="$t('admin.klasse.discard')"
+          id="klasse-creation-form"
+          :onDiscard="navigateToKlasseManagement"
+          @onShowDialogChange="(value: boolean) => (showUnsavedChangesDialog = value)"
+          :onSubmit="onSubmit"
+          :showUnsavedChangesDialog="showUnsavedChangesDialog"
+        >
+          <!-- Organisationsebene -->
+          <v-row>
+            <h3 class="headline-3">1. {{ $t('admin.organisation.assignOrganisation') }}</h3>
+          </v-row>
+          <FormRow
+            :errorLabel="selectedSchuleProps['error']"
+            labelForId="schule-select"
+            :isRequired="true"
+            :label="$t('admin.organisation.organisation')"
+          >
+            <v-autocomplete
+              autocomplete="off"
+              clearable
+              data-testid="schule-select"
+              density="compact"
+              id="schule-select"
+              :items="schulen"
+              item-value="value"
+              item-text="title"
+              :placeholder="$t('admin.organisation.assignOrganisation')"
+              required="true"
+              variant="outlined"
+              v-bind="selectedSchuleProps"
+              v-model="selectedSchule"
+              v-model:search="searchInputSchule"
+              :no-data-text="$t('noDataFound')"
+            ></v-autocomplete>
+          </FormRow>
+
+          <!-- Klassenname -->
+          <v-row>
+            <v-col>
+              <h3 class="headline-3">2. {{ $t('admin.klasse.enterKlassenname') }}</h3>
+            </v-col>
+          </v-row>
+          <FormRow
+            :errorLabel="selectedKlassennameProps['error']"
+            labelForId="klassenname-input"
+            :isRequired="true"
+            :label="$t('admin.klasse.klassenname')"
+          >
+            <v-text-field
+              data-testid="klassenname-input"
+              v-bind="selectedKlassennameProps"
+              v-model="selectedKlassenname"
+              :placeholder="$t('admin.klasse.enterKlassenname')"
+              variant="outlined"
+              density="compact"
+              required
+            ></v-text-field>
+          </FormRow>
+        </FormWrapper>
+      </template>
+
+      <!-- Result template on success after submit  -->
+      <template v-if="organisationStore.createdOrganisation && !organisationStore.errorCode">
+        <v-container class="new-role-success">
```

Comment:
```suggestion
        <v-container class="new-klasse-success">
```
Außerdem wird die Klasse von unseren Styles gar nicht verwendet. Kann also sowohl hier als auch in der PersonCreationView und der RolleCreationView komplett raus

Reply:
Done.